### PR TITLE
[gRNAet7B] Fixes trigger procedures in clusters for neo4j 5.x (extended part)

### PR DESCRIPTION
**Pull request by vga91**
TODO: Waiting for https://github.com/neo4j/apoc/pull/250
Change `.gitmodules` branch to dev when the pr is merged

### CODE REVIEW

**Review by ncordon — CHANGES_REQUESTED** [review]
Probably these tests should be moved to core?

**Comment by vga91** [comment]
> Probably these tests should be moved to core?

They are on `extended` because leverage on extended functions, e.g. `apoc.trigger.nodesByLabel`.

File: .gitmodules [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [submodule "apoc-core"]
 	path = apoc-core
 	url = https://github.com/neo4j/apoc
-	branch = dev
+	branch = dev-fix-trigger-procedures
```
Comment (ncordon) [inline .gitmodules:4]: This should always point to `dev`

**Review by ncordon — COMMENTED** [review]
It looks good, happy to approve once the branch of the submodule has been changed

**Comment by vga91** [comment]
Closed in favor of https://github.com/neo4j-contrib/neo4j-apoc-procedures/pull/3391

**Pull request closed without merging**